packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js

```diff
@@ -38,12 +38,45 @@ export class CheckoutDeliveryOption extends PureComponent {
         optionSubPrice: 0
     };
 
+    state = { isOverflowed: false };
+
+    rowRef = createRef();
+
+    subPriceRef = createRef();
+
+    componentDidMount() {
+        this.updateState();
+        window.addEventListener('resize', () => this.updateState());
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', () => this.updateState());
+    }
+
+    updateState() {
+        const { current: rowEl } = this.rowRef;
+        const { current: subPriceEl } = this.subPriceRef;
+
+        if (rowEl && subPriceEl) {
+            const rowLeft = rowEl.getBoundingClientRect().left;
+            const SubPriceLeft = this.subPriceRef.current.getBoundingClientRect().left;
```

Comment:
rename `SubPriceLeft ` to `subPriceLeft`
change `this.subPriceRef.current.` to `subPriceEl.`